src/main/java/com/google/sps/server/Auth.java

```diff
@@ -33,7 +33,7 @@
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException {
-    response.sendRedirect("/course-form.html");
+    response.sendRedirect("/?signedIn=true");
```

Comment:
While this URL param trick will mostly work ok, it's not reliable. Anyone can go and type that param in the URL bar and the frontend will behave as though the user had signed in when they hadn't.

You can address this in a follow-up PR if it's too big a change for now, here is a suggestion as to the officially supported way to do this: https://stackoverflow.com/a/50256404

Reply:
I evidently should have done more searching about checking the sign in state, but thank you for bringing this to my attention, I really appreciate it and will make sure to be more secure next time. I'm going to open a new PR for this change.